newdle/client/src/components/common/TimezonePicker.js

```diff
@@ -17,7 +18,7 @@ function TimezonePicker({onChange, currentTz, title, ...props}) {
   );
   return (
     <div>
-      {title}
+      <Trans>{title}</Trans>
```

Comment:
This makes no sense, you cannot translate a string that is not known beforehand. This file has no translatable content (which is why there's no issue for it).